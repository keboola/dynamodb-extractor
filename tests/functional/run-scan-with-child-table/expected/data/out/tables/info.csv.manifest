{
  "incremental": true,
  "columns": [
    "rating",
    "10-movies_pk"
  ],
  "column_metadata": {
    "rating": [
      {
        "key": "KBC.datatype.nullable",
        "value": true
      },
      {
        "key": "KBC.datatype.basetype",
        "value": "string"
      }
    ],
    "10-movies_pk": [
      {
        "key": "KBC.datatype.nullable",
        "value": true
      },
      {
        "key": "KBC.datatype.basetype",
        "value": "string"
      }
    ]
  }
}