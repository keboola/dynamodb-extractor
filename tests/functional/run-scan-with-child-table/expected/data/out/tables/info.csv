"","d008089a1cb90697b3bbef6941c29624"
"","f1f4064e7780cc73cefd9bf3e3ec5e5c"
"7.1","0573c5cea38a536e2ea911e1d23ac083"
"7.3","4245e6dcfce5a9275511ec172690977e"
"8.2","9b20dbf9a7f06306ef49368827ffa11b"
"8.3","a214d738e894ec7c6b4137f5265daa9a"
"","ef32318cfef9aa96c9ceb973a7b3a642"
"7.2","94ecb1a4a5b002e7063f78d9b60cd041"
"","7f7968accdac206fed62393066f51388"
"7.1","084592b8967def33d25ef52abcc0b583"
